include/MixerChannelView.h

```diff
@@ -119,6 +123,7 @@ namespace lmms::gui
         QLabel* m_receiveArrow;
         PixmapButton* m_muteButton;
         PixmapButton* m_soloButton;
+        PeakIndicator* m_peakIndicator = 0;
```

Comment:
We prefer `nullptr` over `0` for a null pointer constant.
```suggestion
        PeakIndicator* m_peakIndicator = nullptr;
```

Reply:
Fixed with commit aea000bc594.